library/cluster_vm.py

```diff
@@ -18,7 +18,7 @@
 
 short_description: Manages virtual machines on a SEAPATH cluster
 
-version_added: "2.9"
+version_added: "2.10"
```

Comment:
According to [ansible documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html), this version field should not be changed
Even if compatible with Ansible 2.10, the module was still added with Ansible 2.9